latest/examples/valid_strict/multiscales_example.json

```diff
@@ -0,0 +1,54 @@
+{
+    "multiscales": [
+        {
+            "version": "0.5-dev",
+            "name": "example",
+            "axes": [
+                {"name": "t", "type": "time", "unit": "millisecond"},
+                {"name": "c", "type": "channel"},
+                {"name": "z", "type": "space", "unit": "micrometer"},
+                {"name": "y", "type": "space", "unit": "micrometer"},
+                {"name": "x", "type": "space", "unit": "micrometer"}
+            ],
+            "datasets": [
+                {
+                    "path": "0",
+                    "coordinateTransformations": [{
+                        // the voxel size for the first scale level (0.5 micrometer)
+                        "type": "scale",
+                        "scale": [1.0, 1.0, 0.5, 0.5, 0.5]
+                    }]
+                },
+                {
+                    "path": "1",
+                    "coordinateTransformations": [{
+                        // the voxel size for the second scale level (downscaled by a factor of 2 -> 1 micrometer)
+                        "type": "scale",
+                        "scale": [1.0, 1.0, 1.0, 1.0, 1.0]
+                    }]
+                },
+                {
+                    "path": "2",
+                    "coordinateTransformations": [{
+                        // the voxel size for the third scale level (downscaled by a factor of 4 -> 2 micrometer)
+                        "type": "scale",
+                        "scale": [1.0, 1.0, 2.0, 2.0, 2.0]
+                    }]
+                }
+            ],
+            "coordinateTransformations": [{
+                // the time unit (0.1 milliseconds), which is the same for each scale level
```

Comment:
Are these comments valid JSON? Do they cause any issues? (I see they are the same as this under 0.4/)

Reply:
from https://github.com/ome/ngff/pull/98#issuecomment-1070598704

> a new Document conventions section is added to the specification which ... clarifies the explanation scope of the comments in the JSON snippets